examples/nextjs/pages/index.js

```diff
@@ -1,30 +1,23 @@
-import React, { useState, useEffect } from 'react'
+import React from 'react'
 import { utils as ethersUtils } from 'ethers'
 import { UseWalletProvider, useWallet } from 'use-wallet'
 
 function App() {
-  const [lastError, setLastError] = useState('')
   const wallet = useWallet()
   const blockNumber = wallet.getBlockNumber()
 
-  const activate = async connector => {
-    setLastError('')
-
-    await wallet.connect(connector)
-  }
-
-  useEffect(() => setLastError(wallet.error?.name ?? ''), [wallet])
+  const activate = async connector => await wallet.connect(connector)
```

Comment:
We could also remove the `async` / `await` since we don’t really need to wait for it to finish:

```suggestion
  const activate = connector => {
    wallet.connect(connector)
  }
```